Xamarin.Forms.Core/Shell/IShellSectionController.cs

```diff
@@ -21,8 +21,7 @@ public interface IShellSectionController : IElementController
 
 		void SendInsetChanged(Thickness inset, double tabThickness);
 
-		void SendPopped();
```

Comment:
Isn't this a breaking change? 
We removed `SendPopped()` and `SendPopped(page)` and added new ones.

Reply:
Controllers are meant to be broken right?

Reply:
It's definitely a breaking change if we expect users to subclass `ShellSection`.

Reply:
Subclassing is fine and won't be breaking. 

It's only breaking for people that have used a custom class that's not ShellSection and added IShellSectionController to it. 

This is no different than any of the other controller interfaces we have on Button, Image, etc..

Reply:
That being said :-p Added them back with some attributes